exchanges/huobi/huobi_wrapper.go

```diff
@@ -616,28 +616,55 @@ func (h *HUOBI) GetOrderInfo(orderID string) (order.Detail, error) {
 	if respData.ID == 0 {
 		return orderDetail, fmt.Errorf("%s - order not found for orderid %s", h.Name, orderID)
 	}
-
+	var responseID = strconv.FormatInt(respData.ID, 10)
+	if responseID != orderID {
+		return orderDetail, errors.New(h.Name + " - GetOrderInfo orderID mismatch. Expected: " + orderID + " Received: " + responseID)
+	}
 	typeDetails := strings.Split(respData.Type, "-")
 	orderSide, err := order.StringToOrderSide(typeDetails[0])
 	if err != nil {
-		return orderDetail, err
+		if h.Websocket.IsConnected() {
+			h.Websocket.DataHandler <- order.ClassificationError{
+				Exchange: h.Name,
+				OrderID:  orderID,
+				Err:      err,
+			}
+		} else {
+			return orderDetail, err
+		}
 	}
 	orderType, err := order.StringToOrderType(typeDetails[1])
 	if err != nil {
-		return orderDetail, err
+		if h.Websocket.IsConnected() {
+			h.Websocket.DataHandler <- order.ClassificationError{
+				Exchange: h.Name,
+				OrderID:  orderID,
+				Err:      err,
+			}
+		} else {
+			return orderDetail, err
+		}
 	}
 	orderStatus, err := order.StringToOrderStatus(respData.State)
 	if err != nil {
-		return orderDetail, err
+		if h.Websocket.IsConnected() {
+			h.Websocket.DataHandler <- order.ClassificationError{
+				Exchange: h.Name,
+				OrderID:  orderID,
+				Err:      err,
+			}
+		} else {
+			return orderDetail, err
+		}
 	}
 	orderDetail = order.Detail{
 		Exchange:       h.Name,
-		ID:             strconv.FormatInt(respData.ID, 10),
+		ID:             orderID,
 		AccountID:      strconv.FormatInt(respData.AccountID, 10),
-		CurrencyPair:   currency.NewPairFromString(respData.Symbol),
-		OrderType:      orderType,
-		OrderSide:      orderSide,
-		OrderDate:      time.Unix(0, respData.CreatedAt*int64(time.Millisecond)),
+		Pair:           currency.NewPairFromString(respData.Symbol),
```

Comment:
Out of scope, but please use currency.NewPairFromFormattedPairs here otherwise this will cause pair mismatches

Reply:
Bump, this would be good to do in this PR